Prb.Radiators.Wpf/MainWindow.xaml.cs

```diff
@@ -20,10 +20,159 @@ namespace Prb.Radiators.Wpf
     /// </summary>
     public partial class MainWindow : Window
     {
+
+
+       
+        private object item;
+        private object lblRoomVolume;
+        private int roomVolume;
+        private int totalRadiatorWatts;
+
+        double lengthRoom = double.Parse(txtRoomLength.Text);
+        double widthRoom = double.Parse(txtRoomWidth.Text);
+        double heightRoom = double.Parse(txtRoomHeight.Text);
+
+
         public MainWindow()
         {
             InitializeComponent();
+            InitializeComboboxes();
+            ResetControls();
+
+        }
+
+        private void InitializeComboboxes()
+        {
+          
+        }
+        private void ResetControls()
+        {
+            txtRoomHeight.Clear();
+            txtRoomLength.Clear();
+            txtRoomWidth.Clear();
+            lblRoomVolume.Text = "";
+            lblRoomWatts.Content = "";
+            lstRadiators.Items.Clear();
+            lblTotalWatts.Content = "";
+
+            panTop.IsEnabled = true;
+            panLower.IsEnabled = false;
+        }
+
+        private void CalculateRoomVolume()
+        {
+            double lengthRoom = double.Parse(txtRoomLength.Text);
+            double widthRoom = double.Parse(txtRoomWidth.Text);
+            double heightRoom = double.Parse(txtRoomHeight.Text);
+
+            double volume = lengthRoom * widthRoom * heightRoom / 1000000;
+            double watts = volume * 50;
+
+            lblRoomValue.Content = $"{volume: n2}".ToString();  //Alles afronden naar 0.00
+            lblRoomWatts.Content = watts.ToString();
+
+        }
+
+        private void CalculateRoomWatts()
```

Comment:
Dit is dus de methode die het vermogen moet berekenen (terwijl je dat stiekem ook al deed in de methode die het volume berekende). Houd beiden effectief gescheiden. Gebruik ook hier parameters voor de input (volume) en een return value voor de output (vermogen).